content-src/styles/_mixins.scss

```diff
@@ -17,7 +17,7 @@
 }
 
 @mixin dark-theme-only {
-  [lwt-newtab-brighttext]:not(.force-light-theme) & {
```

Comment:
`force-light-theme` appears to be deprecated.

Reply:
yes with [Enable Dark Mode](https://github.com/mozilla/activity-stream/commit/2225e93564508e4ace4e69f2cd2b735fb3c5ee59#diff-3f95337718772ca7ed0b5acae1358dd0) , we are no longer using force-light-theme and should remove it from rest of the places where as its used.

https://searchfox.org/mozilla-central/search?q=force-light-theme&path=
https://github.com/mozilla/activity-stream/commit/8f3f2f6a4660d8bf557e9823ec2d7e78415775b4

Reply:
@gvn as this is follow up of enabling dark mode,  can we do this cleanup as part of this PR?

Reply:
I would prefer to do that as a separate ticket, but I will file it!

Reply:
sounds good, thanks!